src/components/Comment/index.spec.tsx

```diff
@@ -57,29 +75,29 @@ describe(__filename, () => {
 
   it('lets you set a custom class', () => {
     const className = 'ExampleClass';
-    const root = render({ className });
+    const { root } = render({ beginEdit: true, className });
```

Comment:
Why does it need to have `beginEdit: true` in order to set a custom className?

Reply:
`begin: true` is to make the comment render as a form. Otherwise, since `Comment` component requires an `initialComment` when rendering a read-only comment, I need a few more lines to set up a comment in this test.

I added `begin: true` to make this test the same as before, the since the previous `render` function has default `read-only: false`.